test/unwind.js

```diff
@@ -0,0 +1,74 @@
+var R = require('../source');
+var eq = require('./shared/eq');
+
+
+describe('unwind', function() {
+  var object = { name: 'alice',
```

Comment:
I think we should keep ```object``` as global variable as it is used multiple time and would lead to more unnecessary duplication which will not help anyone. However, we should define ```quiz``` inline as it would be better to understand as you said.
Actually, I disabled my prettier due to some reason, that's why ```name``` was not parallel to others. Will correct that.